Traceback (most recent call last):
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\client.py", line 1107, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\asyncio\base_events.py", line 584, in run_until_complete
    return future.result()
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\client.py", line 852, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\csureshkumar\appdata\local\programs\python\python37\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
txt_labels = np.unique(df['Species'])
lbl = txt_labels[0]
fig,ax = plt.subplots(1,2,figsize=(10,5))
df_data = [None]*len(txt_labels)
img = mpimg.imread('img/lter_penguins.png')
ax[0].imshow(img)
ax[0].axis('off')
color = ['tomato','mediumorchid','seagreen','aqua','black','magenta']
for i,lbl in enumerate(txt_labels):
    df_data[i] = df[df['Species'] ==  lbl]
#     print(df_data[i].columns)
    ax[1].scatter(df_data[i]['Flipper Length (mm)'],df_data[i]['Culmen Length (mm)'],color=color[i])
# ax[1].axis('off')
ax[1].set_xlabel('Flipper Length');
ax[1].set_ylabel('Culmen Length');
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
[1;32m<ipython-input-2-ff1659be8ee6>[0m in [0;36m<module>[1;34m[0m
[0;32m      3[0m [0mfig[0m[1;33m,[0m[0max[0m [1;33m=[0m [0mplt[0m[1;33m.[0m[0msubplots[0m[1;33m([0m[1;36m1[0m[1;33m,[0m[1;36m2[0m[1;33m,[0m[0mfigsize[0m[1;33m=[0m[1;33m([0m[1;36m10[0m[1;33m,[0m[1;36m5[0m[1;33m)[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [0mdf_data[0m [1;33m=[0m [1;33m[[0m[1;32mNone[0m[1;33m][0m[1;33m*[0m[0mlen[0m[1;33m([0m[0mtxt_labels[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 5[1;33m [0mimg[0m [1;33m=[0m [0mmpimg[0m[1;33m.[0m[0mimread[0m[1;33m([0m[1;34m'img/lter_penguins.png'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      6[0m [0max[0m[1;33m[[0m[1;36m0[0m[1;33m][0m[1;33m.[0m[0mimshow[0m[1;33m([0m[0mimg[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      7[0m [0max[0m[1;33m[[0m[1;36m0[0m[1;33m][0m[1;33m.[0m[0maxis[0m[1;33m([0m[1;34m'off'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\AppData\Local\Programs\Python\Python37\lib\site-packages\matplotlib\image.py[0m in [0;36mimread[1;34m(fname, format)[0m
[0;32m   1472[0m             [0mfd[0m [1;33m=[0m [0mBytesIO[0m[1;33m([0m[0mrequest[0m[1;33m.[0m[0murlopen[0m[1;33m([0m[0mfname[0m[1;33m)[0m[1;33m.[0m[0mread[0m[1;33m([0m[1;33m)[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1473[0m             [1;32mreturn[0m [0m_png[0m[1;33m.[0m[0mread_png[0m[1;33m([0m[0mfd[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 1474[1;33m     [1;32mwith[0m [0mcbook[0m[1;33m.[0m[0mopen_file_cm[0m[1;33m([0m[0mfname[0m[1;33m,[0m [1;34m"rb"[0m[1;33m)[0m [1;32mas[0m [0mfile[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m   1475[0m         [1;32mreturn[0m [0m_png[0m[1;33m.[0m[0mread_png[0m[1;33m([0m[0mfile[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1476[0m [1;33m[0m[0m

[1;32m~\AppData\Local\Programs\Python\Python37\lib\contextlib.py[0m in [0;36m__enter__[1;34m(self)[0m
[0;32m    110[0m         [1;32mdel[0m [0mself[0m[1;33m.[0m[0margs[0m[1;33m,[0m [0mself[0m[1;33m.[0m[0mkwds[0m[1;33m,[0m [0mself[0m[1;33m.[0m[0mfunc[0m[1;33m[0m[1;33m[0m[0m
[0;32m    111[0m         [1;32mtry[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 112[1;33m             [1;32mreturn[0m [0mnext[0m[1;33m([0m[0mself[0m[1;33m.[0m[0mgen[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    113[0m         [1;32mexcept[0m [0mStopIteration[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    114[0m             [1;32mraise[0m [0mRuntimeError[0m[1;33m([0m[1;34m"generator didn't yield"[0m[1;33m)[0m [1;32mfrom[0m [1;32mNone[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\AppData\Local\Programs\Python\Python37\lib\site-packages\matplotlib\cbook\__init__.py[0m in [0;36mopen_file_cm[1;34m(path_or_file, mode, encoding)[0m
[0;32m    416[0m [1;32mdef[0m [0mopen_file_cm[0m[1;33m([0m[0mpath_or_file[0m[1;33m,[0m [0mmode[0m[1;33m=[0m[1;34m"r"[0m[1;33m,[0m [0mencoding[0m[1;33m=[0m[1;32mNone[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    417[0m     [1;34mr"""Pass through file objects and context-manage `.PathLike`\s."""[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 418[1;33m     [0mfh[0m[1;33m,[0m [0mopened[0m [1;33m=[0m [0mto_filehandle[0m[1;33m([0m[0mpath_or_file[0m[1;33m,[0m [0mmode[0m[1;33m,[0m [1;32mTrue[0m[1;33m,[0m [0mencoding[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    419[0m     [1;32mif[0m [0mopened[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[0;32m    420[0m         [1;32mwith[0m [0mfh[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\AppData\Local\Programs\Python\Python37\lib\site-packages\matplotlib\cbook\__init__.py[0m in [0;36mto_filehandle[1;34m(fname, flag, return_opened, encoding)[0m
[0;32m    401[0m             [0mfh[0m [1;33m=[0m [0mbz2[0m[1;33m.[0m[0mBZ2File[0m[1;33m([0m[0mfname[0m[1;33m,[0m [0mflag[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m    402[0m         [1;32melse[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 403[1;33m             [0mfh[0m [1;33m=[0m [0mopen[0m[1;33m([0m[0mfname[0m[1;33m,[0m [0mflag[0m[1;33m,[0m [0mencoding[0m[1;33m=[0m[0mencoding[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    404[0m         [0mopened[0m [1;33m=[0m [1;32mTrue[0m[1;33m[0m[1;33m[0m[0m
[0;32m    405[0m     [1;32melif[0m [0mhasattr[0m[1;33m([0m[0mfname[0m[1;33m,[0m [1;34m'seek'[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'img/lter_penguins.png'
FileNotFoundError: [Errno 2] No such file or directory: 'img/lter_penguins.png'

